packages/hash/design-system/src/ui/utils/mui-utils.ts

```diff
@@ -1,13 +1,5 @@
 import { useLayoutEffect } from "react";
 
-export const parseIntFromPixelString = (pixelString: string): number => {
-  if (!pixelString.endsWith("px")) {
-    throw new Error(`The pixel string "${pixelString}" doesn't end with "px"`);
-  }
-
-  return parseInt(pixelString.slice(0, -2), 10);
-};
-
 /**
```

Comment:
I guess we can name this file `use-scroll-lock.ts` because `useScrollLock` is the only exports. We don't seem to need `utils` `mui-utils` nesting any more unless I miss something.